config/modules/solanum/default.nix

```diff
@@ -214,7 +214,8 @@ in {
           };
 
           maxAutoconn = mkOption {
-            type = ints.between 0 1;
+            type = nullOr (ints.between 0 1);
+            default = null;
```

Comment:
Sneaky :)

Reply:
Things like this turn up, when you suddenly have different connection classes than just servers :laughing: